tests/validation/tests/v3_api/common.py

```diff
@@ -940,7 +945,8 @@ def exec_shell_command(ip, port, cmd, password, user="root", sshKey=None):
     ssh = paramiko.SSHClient()
     ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
     if sshKey:
-        ssh.connect(ip, username=user, key_filename=sshKey, port=port)
+        key = paramiko.RSAKey.from_private_key_file(sshKey)
+        ssh.connect(ip, username=user, pkey=key, port=port)
```

Comment:
Is using key_filename not working in this case ? Would prefer not to introduce this change , since this code is being used as part of custom cluster creation and works as expected.

Reply:
We're now using `execute_command` from `Node` class.